modules/nw-kube-proxy-config.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * networking/configuring-kubeproxy.adoc
+
+[id="modifying-kubeproxy-config-{context}"]
+= kube-proxy configuration parameters
+
+You can modify the following `kubeProxyConfig` parameters:
+
+.Parameters
+[cols=".^2,.^5,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`bindAddress`
+|The IP address for `kube-proxy` to serve on.
+|The default value is `0.0.0.0`.
+
+|`iptablesSyncPeriod`
+|The refresh period for `iptables` rules. The default value is `30s`. Valid
+suffixes include `s`, `m`, and `h` and are described in the
+link:https://golang.org/pkg/time/#ParseDuration[Go time package] documentation.
+|`30s`
+
+|`proxyArguments.iptables-min-sync-period`
+|The minimum duration before refreshing `iptables` rules. This parameter ensures
+that the refresh does not happen too frequently. Valid suffixes include `s`,
+`m`, and `h` and are described in the
+link:https://golang.org/pkg/time/#ParseDuration[Go time package]
+|`30s`
+
+|`proxyArguments.proxy-mode`
+|The implmentation for synchronizing network rules.
+|The `proxy-mode` can be either `iptables` or `userspace`. The default value is
+`iptables`.
+
+|====
+
+For a complete list of the parameters that `proxyArguments` supports refer to
```

Comment:
So, are you saying that you can use all of the available proxyArgument parameters? That's not clear.

If you can use all of the proxyArguments, consider splitting the table so that you can provide information about the proxyArgument parameters as a whole, including the link, and then provide examples. If these are required or most important, make that clear.

If they're not all available, why do you need this link?

Reply:
@kalexand-rh, do you mean a separate table that covers only `proxyArguments`?

@squeed, @danwinship: Do you know which `proxyArguments` we want to explicitly expose in the docs? Right now we have `proxy-mode` and `iptables-min-sync-period`. By my count there are _40_ parameters for kube-proxy.

Thanks!

Reply:
@boxman, that depends. If you just have the two, you could complicate the table to cover them. If you need to cover all 40, which I hope you don't need to, use a separate table.

Reply:
We want to document whatever we documented in 3.x. That might be just proxy-mode and iptables-min-sync-period?

I don't think we need to point to the "complete list" of parameters. While it's true that people *can* override all of those options, we definitely don't actually *want* them to...

Reply:
@danwinship, which ones do you *want* them to override? Just the ones from 3.x? If you *can* do it, but we don't want them to, we should include a disclaimer that "while other proxyArguments are available, only changing the specified parameters is supported" or whatever the restriction is.

Reply:
I think changing any of the options is technically supported, it's just not recommended. There are a few that most customers would never need to use, but a few have legitimate use cases for. But there are also some that are mostly just useful for shooting yourself in the foot with, so we don't want to be actively pointing people toward them.

Reply:
Realistically the only one I'd like to see documented is "iptables-min-sync-period". We don't support changing `proxy-mode` anymore. Also, we will be moving away kube-proxy args and start using the kube-proxy configuration object.